hack/make/ubuntu

```diff
@@ -91,6 +96,11 @@ if [ "$1" = 'configure' ] && [ -z "$2" ]; then
 	fi
 fi
 
+if [ -x /sbin/apparmor_parser ]; then
+	/sbin/apparmor_parser -r -W -T /etc/apparmor.d/ docker
+	/sbin/apparmor_parser -r -W -T /etc/apparmor.d/ docker-engine
+fi
```

Comment:
so I think the easiest way is to also add these lines in hack/make/build-deb or hack/make/.build-deb/{files}